esmvaltool/cmorizers/data/downloaders/datasets/esacci_soil_moisture.py

```diff
@@ -0,0 +1,45 @@
+"""Script to download ESACCI-SOIL-MOISTURE."""
```

Comment:
In `datasets.yml` this dataset is listed under `ESACCI-SOILMOISTURE`. It would be good to keep only one format throughout this PR: SOIL-MOISTURE or SOILMOISTURE. with the right dataset name, this script works fine

Reply:
Now it is consistent